backend/tests/query_benchmark/conftest.py

```diff
@@ -1,44 +1,64 @@
+from pathlib import Path
+from typing import Any
+
 import pytest
 
+from infrahub.core.constants import BranchSupportType
 from infrahub.core.schema import SchemaRoot
-from tests.helpers.query_benchmark.db_query_profiler import QueryAnalyzer
-
 
-@pytest.fixture(scope="session")
-def query_analyzer() -> QueryAnalyzer:
-    return QueryAnalyzer()
+RESULTS_FOLDER = Path(__file__).resolve().parent / "query_performance_results"
 
 
 @pytest.fixture
-def car_person_schema_root() -> SchemaRoot:
-    schema = {
+async def car_person_schema_root() -> SchemaRoot:
+    schema: dict[str, Any] = {
         "nodes": [
             {
                 "name": "Car",
                 "namespace": "Test",
                 "default_filter": "name__value",
-                "display_labels": ["name__value", "nbr_seats__value"],
+                "display_labels": ["name__value", "color__value"],
+                "uniqueness_constraints": [["name__value"]],
+                "branch": BranchSupportType.AWARE.value,
                 "attributes": [
                     {"name": "name", "kind": "Text", "unique": True},
-                    {"name": "nbr_seats", "kind": "Number"},
+                    {"name": "nbr_seats", "kind": "Number", "optional": True},
+                    {"name": "color", "kind": "Text", "default_value": "#444444", "max_length": 7, "optional": True},
+                    {"name": "is_electric", "kind": "Boolean", "optional": True},
+                    {
+                        "name": "transmission",
+                        "kind": "Text",
+                        "optional": True,
+                        "enum": ["manual", "automatic", "flintstone-feet"],
+                    },
                 ],
                 "relationships": [
-                    {"name": "owner", "peer": "TestPerson", "optional": True, "cardinality": "one"},
+                    {
+                        "name": "owner",
+                        "label": "Commander of Car",
+                        "peer": "TestPerson",
+                        "cardinality": "one",
+                        "direction": "outbound",
```

Comment:
unless there is a specific reason to set the direction, It's usually best to keep the default